backend/src/eml/base.rs

```diff
@@ -79,13 +79,38 @@ pub trait EMLDocument: Sized + DeserializeOwned + Serialize {
         let serializer = {
             let mut s = Serializer::with_root(writer, Some("EML"))
                 .expect("Constructing a new XML Serializer cannot fail");
-            s.indent(' ', 4);
+            s.indent(' ', 2);
             s
         };
         Serialize::serialize(&self, serializer)
     }
 }
 
+pub fn eml_document_hash(input: &str, chunked: bool) -> String {
```

Comment:
I think it's good to test this function, e.g.
```rust
    #[test]
    fn test_eml_document_hash() {
        let input = "test";
        let hash = eml_document_hash(input, false);
        assert_eq!(
            hash,
            "9F86D081884C7D659A2FEAA0C55AD015A3BF4F1B2B0B822CD15D6C15B0F00A08"
        );
        let hash = eml_document_hash(input, true);
        assert_eq!(
            hash,
            "9F86 D081 884C 7D65 9A2F EAA0 C55A D015 A3BF 4F1B 2B0B 822C D15D 6C15 B0F0 0A08"
        );
    }
```

Reply:
I copied your suggestion into a1ab3de98bb3c9e900a1e9af1e0ed65bb74a4f33 :)